docs/api/app.md

```diff
@@ -573,3 +573,31 @@ Hooks is a method to return [hooks](./hooks.md) property.
 ```go title="Signature"
 func (app *App) Hooks() *Hooks
 ```
+
+## RebuildTree
```

Comment:
Can you update what's new guide too

Reply:
I think I have updated it in the right way. It's worth the review tho